app/src/main/java/org/schabi/newpipe/fragments/detail/VideoDetailFragment.java

```diff
@@ -1040,8 +1038,8 @@ private void showContentWithAnimation(long duration,
         contentRootLayoutHiding.animate()
                 .alpha(1f)
                 .translationY(0)
-                .setStartDelay(delay)
-                .setDuration(duration)
+                .setStartDelay((long) 0)
```

Comment:
Delay & Duration should be declared somewhere, its not good to have these values multiple times as inline within the code.